src/schema/v2/infiniteDiscovery/discoverArtworks.ts

```diff
@@ -179,55 +179,96 @@ export const DiscoverArtworks: GraphQLFieldConfig<void, ResolverContext> = {
       user.seenArtworks?.map((node) => node.internalID) || []
     const artworkIdsToFilter = [...likedArtworkIds, ...seenArtworkIds]
 
-    let searchQuery
-
+    let response
     if (likedArtworkIds.length > 0) {
```

Comment:
**suggestion:** Rather than declaring response here, and later mutating the response we could instead, get all data (in both cases we'd need curated artworks), and then generate the shuffled response based on this case. Similar to [this](https://github.com/artsy/metaphysics/blob/be271119240cf778fb1cdd1836789bbac73dbf41/src/schema/v2/infiniteDiscovery/discoverArtworks.ts#L113)